PWGJE/EMCALJetTasks/AliEmcalJetTask.cxx

```diff
@@ -274,6 +282,7 @@ Int_t AliEmcalJetTask::FindJets()
       if (fApplyArtificialTrackingEfficiency) {
         if (fTrackEfficiencyOnlyForEmbedding == kFALSE || (fTrackEfficiencyOnlyForEmbedding == kTRUE && tracks->GetIsEmbedding())) {
           Double_t trackEfficiency = fTrackEfficiencyFunction->Eval(it->first.Pt());
+          if(fApplyTrackingEfficiencyFromHistogram) trackEfficiency -= (1. - fTrackEfficiencyHistogram->GetBinContent(fTrackEfficiencyHistogram->FindBin(it->first.Pt())));
```

Comment:
Please wrap the statement in `{` `}`. I know it's exactly the same, but it's easy to introduce bugs without the explicit wrapping (I know that it's not consistent - sorry to be insist)

Reply:
done